precompiles/staking/staking_test.go

```diff
@@ -96,27 +96,27 @@ func (s *PrecompileTestSuite) TestRequiredGas() {
 				input, err := s.precompile.Pack(
 					staking.DelegateMethod,
 					s.address,
-					s.validators[0].GetOperator().String(),
+					common.BytesToAddress(s.validators[0].GetOperator().Bytes()),
 					big.NewInt(10000000000),
 				)
 				s.Require().NoError(err)
 				return input
 			},
-			7760,
+			4880,
 		},
 		{
 			"success - undelegate transaction with correct gas estimation",
 			func() []byte {
 				input, err := s.precompile.Pack(
 					staking.UndelegateMethod,
 					s.address,
-					s.validators[0].GetOperator().String(),
+					common.BytesToAddress(s.validators[0].GetOperator().Bytes()),
```

Comment:
We need to a typescript library to convert from validator bech32 to hex in order to update the frontend. Without the lib the frontend will not be able to create staking transactions after upgrading the network with the new code.